examples/rapids/02-rapids-gpu-cluster.ipynb

```diff
@@ -0,0 +1,542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-10-26T17:00:38.727567Z",
+     "iopub.status.busy": "2021-10-26T17:00:38.727285Z",
+     "iopub.status.idle": "2021-10-26T17:00:38.730682Z",
+     "shell.execute_reply": "2021-10-26T17:00:38.730051Z",
+     "shell.execute_reply.started": "2021-10-26T17:00:38.727541Z"
+    }
+   },
+   "source": [
+    "# Use RAPIDS on a GPU Cluster"
```

Comment:
I think we should assume that people have read the RAPIDs on a single GPU example first, or if they haven't tell them to skim it before they read this one. I would rather do that and avoid having a lot of redundant text.